packages/upload-api/src/types/usage.ts

```diff
@@ -13,4 +15,10 @@ export interface UsageStorage {
     space: SpaceDID,
     period: { from: Date; to: Date }
   ) => Promise<Result<UsageData, Failure>>
+  record: (
+    customer: AccountDID,
+    resource: UnknownLink,
+    bytes: number,
+    servedAt: Date
```

Comment:
We should add a `cause: Link` field to identify the invocation.

Reply:
@alanshaw Where does it come from?

Reply:
In a ucanto handler it is `invocation.cid`